Makefile

```diff
@@ -26,7 +26,8 @@ generate-test: manifests
 
 # Generate manifests e.g. CRD, RBAC etc.
 manifests:
-	go run vendor/sigs.k8s.io/controller-tools/cmd/controller-gen/main.go all
+	go run vendor/sigs.k8s.io/controller-tools/cmd/controller-gen/main.go crd --output-dir config/default/crd
```

Comment:
could you please comment on this change in the commit message? or preferably move it to a separate commit?